cypress/integration/userCanSeeFirstHeader.feature.js

```diff
@@ -0,0 +1,9 @@
+describe("user views menus", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3001");
```

Comment:
If you are using base Url as (http://localhost:3001)

test should be: cy.visit("/")   ??

Reply:
Fixed.